src/TwilioConfig.php

```diff
@@ -92,4 +94,21 @@ public function getServiceSid()
             return $this->config['sms_service_sid'];
         }
     }
+
+    /**
+     * Returns list of codes for Twilio errors, which should be ignored (suppressed).
+     *
+     * @see https://www.twilio.com/docs/api/errors
+     *
+     * @return array
+     */
+    public function getIgnoredErrorCodes()
+    {
+        return Arr::get($this->config, 'ignored_error_codes', [
+            21608,
```

Comment:
Could we set these defaults in the config file (and add comments to clarify what each one is)?

Reply:
This extension does not provide pre-defined config file.
Developer should manually add 'twilio' section into 'config/services.php'.
Thus such solution is preferrable.

You can find all error codes and thier meaning at https://www.twilio.com/docs/api/errors, which is linked in PHPDoc here.

Reply:
I think if we're going with config this complex, we should move to a dedicated config file.
Will require a major version bump (unless we implement some kind of fallback to the old services behavior), but I think it's worth it.

Reply:
I think we should also be documenting this in the readme as well.

Reply:
In case you are going to make an overhauling for whole package, there is no sense in polishing this particular PR.